### ISSUE TITLE
HDS Link Detector doesn't work on NBC Olympics

### ISSUE THREAD

**Issue by yjinbhhs** [K-S-V/Scripts#86]
Hi, I downloaded HDS Link Detector Add-on for Firefox (https://addons.mozilla.org/en-US/firefox/addon/hds-link-detector/). But it doesn't work on some of the NBC Olympics link, for example:

http://stream.nbcolympics.com/nbc-day-1-primetime
http://stream.nbcolympics.com/nbc-day-7-primetime

But it works for day-2, 3, 4, 5 and 6. What is the reason behind this? Thanks!

Yan

**Comment by xlhdh** [comment]
If you have access to the browser cache, look for an auth string starting with hdntl. Use that auth string in both manifest and auth instead of the one provided by the add-on.

**Comment by K-S-V** [comment]
Personally, I don't have access to NBC website to test those links myself so I can't say for sure but it's quite possible that those links may not be using HDS at all.

**Comment by yjinbhhs** [comment]
They do use HDS. I actually downloaded some of the files with AdobeHDS (14 out of 16), but there are two of them that I can't. These two may not use HDS. I shared the cache file from Firefox here (I cleared the cache before I clicked that link). Could any of you tell me what technique they use for these two streams? It is kinda of pity that I can't get the complete set... I would really appreciate if anybody can help me out!

https://drive.google.com/file/d/0B-Tw8d5DleQdX2RnalRHNTYwQk0/view?usp=sharing


[Cache_Storage_Information.pdf](https://github.com/K-S-V/Scripts/files/431598/Cache_Storage_Information.pdf)
